package.json

```
@@ -15,11 +15,13 @@
     "cypress-postreport": "yarn cypress-merge && yarn cypress-generate",
     "test-cypress": "node_modules/.bin/cypress open --config-file ./cypress/cypress.json --env openshift=true",
     "test-cypress-headless": "node --max-old-space-size=4096 node_modules/.bin/cypress run --config-file ./cypress/cypress.json --env openshift=true --browser ${BRIDGE_E2E_BROWSER_NAME:=chrome} --headless",
-    "build": "yarn clean && NODE_ENV=production I8N_NS=plugin__odf-console yarn ts-node ./node_modules/.bin/webpack",
-    "build-mco": "yarn clean && NODE_ENV=production MODE=MCO I8N_NS=plugin__odf-multicluster-console yarn ts-node ./node_modules/.bin/webpack",
-    "build-dev": "yarn clean && yarn ts-node ./node_modules/.bin/webpack",
-    "dev": "yarn clean &&  I8N_NS=plugin__odf-console yarn ts-node ./node_modules/.bin/webpack serve --progress",
-    "dev-mco": "yarn clean && MODE=MCO I8N_NS=plugin__odf-multicluster-console yarn ts-node ./node_modules/.bin/webpack serve",
+    "build:plugin": "cd plugins/${PLUGIN} && ts-node ../../scripts/generatePluginPackage.ts && yarn clean && NODE_ENV={MODE} I8N_NS=${I8N_NS} yarn ts-node ../../node_modules/.bin/webpack -c ../../webpack.config.ts",
+    "build": "NODE_ENV=production PLUGIN=odf I8N_NS=plugin__odf-console yarn build:plugin",
+    "build-mco": "NODE_ENV=production PLUGIN=mco I8N_NS=plugin__odf-multicluster-console yarn build:plugin",
+    "build-dev": "NODE_ENV=production PLUGIN=odf I8N_NS=plugin__odf-console yarn build:plugin",
+    "server:plugin": "cd plugins/${PLUGIN} && ts-node ../../scripts/generatePluginPackage.ts && yarn clean && NODE_ENV={MODE} I8N_NS=${I8N_NS} yarn ts-node ../../node_modules/.bin/webpack serve -c ../../webpack.config.ts",
```

```suggestion
    "server:plugin": "cd plugins/${PLUGIN} && ts-node ../../scripts/generatePluginPackage.ts && yarn clean && NODE_ENV={MODE} I8N_NS=${I8N_NS} yarn ts-node ../../node_modules/.bin/webpack serve",
```
Webpack should recursively look for the config file? Is it not able to find it?

---

it is not finding, becuase not it is invoked from plugins directory, not root directory

---

I tried, but I am getting errors


Can't resolve './src' in '/home/gshanmug/Workspace/Openshift/odf-console/plugins/mco
Can't resolve './src' in '/home/gshanmug/Workspace/Openshift/odf-console/plugins/odf